tests/vmi_multiqueue_test.go

```diff
@@ -60,26 +59,30 @@ var _ = Describe("[sig-compute]MultiQueue", func() {
 			availableCPUs = tests.GetHighestCPUNumberAmongNodes(virtClient)
 		})
 
-		It("[test_id:4599]should be able to successfully boot fedora to the login prompt with networking mutiqueues enabled without being blocked by selinux", func() {
-			vmi := tests.NewRandomFedoraVMIWithGuestAgent()
-			numCpus := 3
-			Expect(numCpus).To(BeNumerically("<=", availableCPUs),
-				fmt.Sprintf("Testing environment only has nodes with %d CPUs available, but required are %d CPUs", availableCPUs, numCpus),
-			)
-			cpuReq := resource.MustParse(fmt.Sprintf("%d", numCpus))
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = cpuReq
-			multiQueue := true
-			vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &multiQueue
-			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
-
-			By("Creating and starting the VMI")
-			vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-			Expect(err).ToNot(HaveOccurred())
-			tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 360)
-
-			By("Checking if we can login")
-			Expect(libnet.WithIPv6(console.LoginToFedora)(vmi)).To(Succeed())
-		})
+		DescribeTable("should be able to successfully boot fedora to the login prompt with networking mutiqueues enabled without being blocked by selinux",
```

Comment:
I know this is not related, but we do add a test here: What selinux is mentioned here exactly? The one on the node? The one on the guest?

Reply:
The one on the guest, please see here for more info
https://github.com/kubevirt/kubevirt/pull/2941

Reply:
Right, figured afterwards
Ok will remove the new test then, as MQ is already tested with the explicit added interface